acquia_lift.admin.wizard.inc

```diff
@@ -582,21 +625,49 @@ function acquia_lift_personalize_campaign_wizard_review_alter(&$form, &$form_sta
 
     // Add audiences to the summary section.
   $agent_data = $form['#agent'];
+  if (isset($agent_data->data['variation_set_handling']) && $agent_data->data['variation_set_handling'] == ACQUIA_LIFT_DECISION_MULTIVARIATE) {
+    $form['review']['summary_column']['summary']['option_sets'] = array(
+      '#markup' => '<div class="personalize-wizard-summary-item clearfix"><span class="personalize-wizard-summary-count">1</span><div class="personalize-wizard-summary-details"><div class="item-list"><h3>Multivariate Test</h3></div></div></div>'
```

Comment:
This should use the personalize_wizard_summary_count theme function for the markup.   Since there isn't a list - it can just do it the same way that the audiences do further below in this same function.